app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -53,7 +75,7 @@ def create_event_from_string_date(type, *from_keys)
   def create_events_for_tracked_items
     events = []
     TRACKED_ITEM_FIELDS.each do |field|
-      list_objects_with_key(['claimTrackedItems', field], ['openedDate']).each do |obj|
+      sub_objects_with_key_present(['claimTrackedItems', field], ['openedDate']).each do |obj|
```

Comment:
Looking at this a bit more closely, I don't think this returns objects that have `receivedDate` set but `openedDate` is not present, but it should return objects that have one of the attributes present. Can we fix that here and write a test that ensures the correct events are present?